### DISCUSSION TITLE
Processing Microsoft's Global Buildings Dataset

### DISCUSSION THREAD

**Discussion by njuedominic** [microsoft/PlanetaryComputer#303] [General]
Hello All,
I would like to download Microsoft's Global Building dataset for each country and calculate the built-up area in square meters. Computationally, this is resource-intensive on a normal laptop, so I would like to maybe make use of the cloud tools.
Any suggestions on how I go about this? I am pretty new to the cloud environment, but I do have experience with Python.

**Comment by TomAugspurger** [comment]
Thanks for reaching out.

For downloading the data, you can​ use a tool like azcopy.<https://learn.microsoft.com/en-us/azure/storage/common/storage-use-azcopy-v10> That said, we typically don't recommend downloading entire datasets. For very large datasets, or datasets that are updating rapidly this might not be possible.

Instead, we recommend placing your compute in the same Azure region as the data (West Europe, in this case). Azure has a bunch of compute options. https://planetarycomputer.microsoft.com/docs/concepts/computing has an overview and links to more resources.
________________________________
From: Dominic Njue ***@***.***>
Sent: Sunday, December 10, 2023 10:12 PM
To: microsoft/PlanetaryComputer ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [microsoft/PlanetaryComputer] Processing Microsoft's Global Buildings Dataset (Discussion #303)


Hello All,
I would like to download Microsoft's Global Building dataset for each country and calculate the built-up area in square meters. Computationally, this is resource-intensive on a normal laptop, so I would like to maybe make use of the cloud tools.
Any suggestions on how I go about this? I am pretty new to the cloud environment, but I do have experience with Python.

**Reply by njuedominic** [reply]
Thank you, Tom.
